api/src/lib/interface.ts

```diff
@@ -349,14 +349,6 @@ export interface ConfirmVerificationCodeBody {
 const prismaGeneratedFlagType = Prisma.validator<Prisma.PublicationFlagsDefaultArgs>()({});
 export type Flag = Prisma.PublicationFlagsGetPayload<typeof prismaGeneratedFlagType>;
 
-export type FlagCategory =
-    | 'PLAGIARISM'
```

Comment:
This type was unnecessary because we can just use the one from the prisma client.